src/collections/_documentation/performance-monitoring/configuration/react.md

```diff
@@ -0,0 +1,90 @@
+To get started with performance monitoring using Sentry's React SDK, first install the `@sentry/react` and `@sentry/apm` packages:
+
+```bash
+# Using yarn
+$ yarn add @sentry/react @sentry/apm
+
+# Using npm
+$ npm install @sentry/react @sentry/apm
+```
+
+Next, initialize the integration in your call to `Sentry.init`. Make sure this happens before you mount your React component on the page.
+
+```jsx
+import * as Sentry from '@sentry/react';
+import { Integrations } from "@sentry/apm";
+Sentry.init({
+  dsn: '"___PUBLIC_DSN___"',
+  release: 'my-project-name@' + process.env.npm_package_version,
+  integrations: [
+    new Integrations.Tracing(),
+  ],
+  tracesSampleRate: 0.25, // must be present and non-zero
+});
+
+// ...
+
+ReactDOM.render(<App />, rootNode);
+
+// Can also use with React Concurrent Mode
+// ReactDOM.createRoot(rootNode).render(<App />);
+```
+
+### `withProfiler` Higher-Order Component
+
+`@sentry/react` exports a `withProfiler` higher-order component that leverages the `@sentry/apm` Tracing integration to add React related spans to transactions. If the Tracing integration is not enabled, the Profiler will not work.
+
+In the example below, the `withProfiler` higher-order component is used to instrument an App component.
+
+```jsx
+import React from 'react';
+import * as Sentry from '@sentry/react';
+
+class App extends React.Component {
+  render() {
+    return (
+      <FancyComponent>
+        <NestedComponent someProp={2} />
+        <AnotherComponent />
+      </FancyComponent>
+    )
+  }
+}
+
+export default Sentry.withProfiler(App);
+```
+
+The React Profiler current generates spans with three different kinds of op codes: `react.mount`, `react.render`, and `react.update`.
+
+1. `react.mount` is the span that represents how long it took for the profiled component to mount.
```

Comment:
```suggestion
`react.mount`

: The span that represents how long it took for the profiled component to mount.
```

Reply:
Looks like you're defining these op-codes. There's a stylistic way Docs uses to define terminology, components, etc. Trying to be more consistent about it. 😄 Made the changes for ya.

Reply:
Ok great, thanks for pointing that out, I'll start following that convention.